### PR TITLE
Cherry.spoon - a Pomodoro Timer in the menubar

### PR DESCRIPTION
**Pull request by doesnotexist**
Based off the original code in this repository:
https://github.com/dbmrq/dotfiles/blob/master/home/.hammerspoon/cherry.lua

I fixed a minor bug with menuItems disappearing and turned it into a Spoon.  Also I notified the original author @dbmrq who approved this submission to the official repository.   Have tested it locally and works for me.  First time writing a Spoon and actually any lua at all so please let me know if you have any improvements or suggestions. Thanks.

### CODE REVIEW

**Comment by doesnotexist** [comment]
The check is failing but due to malformed docstrings in an unrelated Spoon (WindowSigils).

File: Source/Cherry.spoon/init.lua
Comment (dbmrq) [inline Source/Cherry.spoon/init.lua:14]:
```suggestion
obj.author = "Daniel Marques <danielbmarques@gmail.com> and Omar El-Domeiri <omar@doesnotexist.com>"
```

Great job, thanks! 🍻

**Review by dbmrq — CHANGES_REQUESTED** [review]

**Comment by doesnotexist** [comment]
That last commit to fix doc strings did not accomplish what I had intended.  I can't test the docstrings generation locally, so won't fiddle with that anymore.  Probably should drop that last commit before merging pull request and leave that to main devs.

**Comment by eraserhd** [comment]
@doesnotexist  It's weird because I had trouble with docstrings when I submitted WindowSigils, and I tweaked it to make them pass, and here we are again. :(

**Comment by cmsj** [comment]
Sorry about the docstrings churn, I'm trying to evolve the linter to get ever stricter and it can be painful at times.

**Comment by cmsj** [comment]
Thanks for this, I'll be merging it via #223 to keep the addition of Cherry.spoon isolated to a single merge commit. I pulled in the MouseCircle.spoon fix via 265fc5b, and the WindowSigil docstrings were fixed in b3e9ff3385b96fbdc379dc2645a36bbd31a5a2e5

**Pull request closed without merging**
